cmd/minikube/cmd/config/config.go

```diff
@@ -114,6 +116,10 @@ var settings = []Setting{
 		set:  SetString,
 	},
 	{
+		name: Bootstrapper,
+		set:  SetString, //TODO(r2d4): more validation here?
```

Comment:
nit: This could check a set of valid bootstrappers. It's already verified in GetClusterBootstrapper.